packages/electrode-archetype-react-app/arch-clap.js

```diff
@@ -142,6 +142,17 @@ function removeLogFiles() {
   } catch (e) {} // eslint-disable-line
 }
 
+function setWebpackProfile(profile) {
+  process.env.ELECTRODE_WEBPACK_PROFILE = profile || "production";
+  logger.info(`Electrode Webpack Profile set to ${process.env.ELECTRODE_WEBPACK_PROFILE}`);
+}
+
+function checkForCustomWebpackConfig(defaultFile) {
```

Comment:
shouldn't this use `util/check-custom-config.js`?

Reply:
the utils is for `electrode-archetype-react-app-dev` package.

Reply:
check-custom-config is for `electrode-archetype-react-app` package